src/button/button.style.js

```diff
@@ -1,38 +1,71 @@
 import styled, { css } from "styled-components";
 
-const baseButton = css`
+const baseStyles = css`
   display: flex;
   align-items: center;
   justify-content: space-evenly;
-  font-size: 1.125em;
+  cursor: pointer;
+  padding-left: 10px;
```

Comment:
actually instead of hardcoding these padding values, you use use flex box to handle the layout - if you need to you can change the content justifying to a different layer then add padding to the text or icons. It's just really hard to cover all permutations with padding.

Reply:
yeah I will go for it

Reply:
you are right we can add padding to the children of the component itself. I was following the default button styling and that includes padding on the left and right. 
I guess for just very long text etc etc. I will remove the paddings now

Reply:
![image](https://user-images.githubusercontent.com/4181674/46204714-7e49c880-c31e-11e8-8813-6bb2c0cb3ad4.png)
Just let me know what are the requirements and I will go for it. I am just explaining why padding was there